WISE-Community/WISE-Client
# feat(Show Group Work): Show component work in Grading Tool

## Changes

- Created ShowGroupWorkDisplayComponent that is used by ShowGroupWorkStudent and ShowGroupWorkGrading
- We now Show Group Work in the Grading Tool
- Do not show "Team has not saved any work" for "Show Group Work" components in the Grading Tool

## Test

- Test with https://github.com/WISE-Community/WISE-API/pull/219
- Have two students that are in the same Peer Group submit work
- Have a Show Group Work component show the work from the students
- Make sure the Show Group Work displays properly in the VLE
- Make sure the Show Group Work displays properly in the Grading Tool

Closes #1037
